### ISSUE TITLE
Device protocol PT502 input/output

### ISSUE THREAD

**Issue by nebiti** [traccar/traccar#4670]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

Hi on PT502,PT600 Device protocol, there are 4 inputs and 4 outputs. but parsed data from the device shows only 1 input available on attribute.
HEX DATA: 24504f532c303030303030312c3131323734322e3030302c412c303930302e393035302c4e2c30333834352e303630392c452c31332e31322c3231322e33342c3239303432312c2c2c412f30303031302c30303030302f302c302c302c302f3235303136313330302f2f6664372f2f

[Bofan GPS tracker Protocol.pdf](https://github.com/traccar/traccar/files/6398266/Bofan.GPS.tracker.Protocol.pdf)

**Comment by tananaev** [comment]
Feel free to contribute a solution.

**Comment by nebiti** [comment]
am not good at Java, what can be suggested is to get all available IO AVL data from devices based on their protocol and store them to tc_positions attributes and those IO attributes can be set from web platform side. i.e
attribute -   Name      - Type 
DIN1       -   Ignition   - Boolean/ Logical
DIN2       -   SOS         - Boolean/ Logical
AIN         - Analog      - String

**Comment by tananaev** [comment]
You would have to at least provide protocol documentation.

**Comment by nebiti** [comment]
protocol documentation is already attached sir.

**Comment by tananaev** [comment]
This is what I see in the document:
```
IN1: SOS
IN2: KEY2 ALARM
IN3: KEY3 ALARM
```
Doesn't match what you said in previous comment.

**Comment by nebiti** [comment]
yes you are right. i was just putting examples. my point is i couldn't find this IN1,IN2,IN3 parameters in attributes.
